File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/service/repository/AppDatabase.java
package com.antonio.samir.meteoritelandingsspots.service.repository;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

import com.antonio.samir.meteoritelandingsspots.model.Address;
import com.antonio.samir.meteoritelandingsspots.model.Meteorite;

@Database(entities = {Meteorite.class, Address.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract AddressDao addressDao();

    public abstract MeteoriteDao meteoriteDao();

}

File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/service/repository/local/database/AppDataBase.kt
package com.antonio.samir.meteoritelandingsspots.service.repository.local.database

import androidx.room.Database
import androidx.room.RoomDatabase
import com.antonio.samir.meteoritelandingsspots.service.business.model.Meteorite

@Database(entities = arrayOf(Meteorite::class), version = 3)
abstract class AppDataBase : RoomDatabase() {

    abstract fun meteoriteDao(): MeteoriteDao

}
